### PR TITLE
Registration bug

### PR DESCRIPTION
**Pull request by afletch24**
The meeting options bug has been resolved and has been converted into redux. The bug was in the registration form for a new user and would occur when the user clicked a meeting option from the drop down menu. The user can now click one or multiple options successfully. 

Initially, the allMeetingOptions array was undefined. After binding the value and pulling the data from props, the error was resolved. 

https://trello.com/c/TaKLYNBu/138-as-a-user-i-am-unable-to-successfully-register-for-the-app

### CODE REVIEW

File: src/LoginPage/components/RegisterForm.js [hunk 1]
```diff
@@ -21,36 +22,33 @@ import "../css/register.css";
...
+    return {allMeetingOptions}
+}
 class RegisterForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
       user: {},
       imgURL:
-        "https://image.freepik.com/free-icon/female-student-silhouette_318-62252.jpg"
+        "https://image.freepik.com/free-icon/female-student-silhouette_318-62252.jpg",
```
Comment (mkarazin) [inline src/LoginPage/components/RegisterForm.js]: Is this a good candidate to either place within the project as an asset, or to use a configuration setting to link to the image. I'd recommend not linking directly to a resource which may change.

File: src/reducers/meeting_options_reducer.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { FETCH_MEETING_OPTIONS } from '../actions/types';
+
+const INITIAL_STATE = [];
+export default (state = INITIAL_STATE, action) => {
+    switch(action.type){
+        case FETCH_MEETING_OPTIONS:
+            return action.payload.data;
```
Comment (mkarazin) [inline src/reducers/meeting_options_reducer.js]: I may be off base here. But in general, I believe that reducers should not set state directly. Instead it should use overwrite portions of state. Such as:
```
return {
   ...state,
   ...action.payload.data,
};
```

File: src/LoginPage/components/RegisterForm.js [hunk 3]
```diff
@@ -21,36 +22,33 @@ import "../css/register.css";
 import createFragment from "react-addons-create-fragment";
 import { ValidatorComponent } from 'react-material-ui-form-validator';
 import holder from "../images/holder.jpg";
+import { fetchMeetingOptions } from "../../actions";
 
+function mapStateToProps ({ allMeetingOptions }){
+    return {allMeetingOptions}
```
Comment (mkarazin) [inline src/LoginPage/components/RegisterForm.js]: For readability, I'd recommend using a consistent styling of object literals and spacing. The standard is typically using spacing.
e.g. `{ allMeetingOptions }` versus `{allMeetingOptions}`.

https://eslint.org/docs/rules/object-curly-spacing

File: src/LoginPage/components/RegisterForm.js (hunk 1 above)
Comment (tejas-rawal) [reply src/LoginPage/components/RegisterForm.js]: Yeah, I would suggest the same thing. We shouldn't be requesting images from the internet. We can add it to the public directory and reference it from there.

File: src/reducers/meeting_options_reducer.js (hunk 2 above)
Comment (tejas-rawal) [reply src/reducers/meeting_options_reducer.js]: Good catch!

**Pull request merged**